indico/web/client/styles/partials/_main.scss

```diff
@@ -11,26 +11,63 @@
 
 /* Styles for the main menu */
 
-.toolbar.global-menu {
-  @include font-family-title-light();
+#global-menu {
+  // FIXME: This should be in the global reset
 
-  height: 3em;
-  line-height: 3em;
+  * {
+    box-sizing: border-box;
+  }
```

Comment:
I think this is no longer needed since you added it globally in a previous PR?

Reply:
I'm going to clean this up later once I get done with category display page. Still have some issues to address there. But yeah, I'll remove this once I get it it.